dts/bindings/dac/nxp, lpdac.yaml

```diff
@@ -0,0 +1,29 @@
+ # Copyright 2023, NXP
+ # All rights reserved.
+ #
+ # SPDX-License-Identifier: BSD-3-Clause
+
+description: NXP MCUX LPDAC
+
+compatible: "nxp,lpdac"
+
+include: dac-controller.yaml
+
+properties:
+  reg:
+    required: true
+
+  voltage-reference:
+    type: int
+    required: true
+    description: DAC voltage reference select
```

Comment:
I believe that the meaning of this value is SOC specific- could we add something to the description here clarifying that, just to avoid ambiguity?